eisbuk-admin/src/components/atoms/AttendanceCard/AttendanceCard.tsx

```diff
@@ -36,28 +55,65 @@ const AttendanceCard: React.FC<Props> = ({
   const endTime = luxonEnd.toISOTime().substring(0, 5);
 
   const timeString = `${startTime} - ${endTime}`;
+
   return (
-    <div>
-      <div data-testid="time-string">{timeString}</div>
-      <div>{categories}</div>
-      {userBookings.map((user) => {
-        const isAbsent = absentees?.includes(user.customer_id) || false;
-        return (
-          <div key={user.customer_id}>
-            <div>{user.name}</div>
-            <button
-              type="button"
-              onClick={() =>
-                dispatch(markAttendance(user.customer_id, isAbsent))
+    <Container maxWidth="sm">
+      <List className={classes.root}>
+        <div key={id + "-" + timeString} className={classes.slotWrapper}>
+          <ListItem className={classes.listHeader}>
+            <ListItemText
+              primary={
+                <span>
+                  {timeString} <b>({userBookings.length})</b>
+                </span>
               }
-            >
-              {isAbsent ? "👎" : "👍"}
-            </button>
-          </div>
-        );
-      })}
-    </div>
+              secondary={translateAndJoinTags(categories, type)}
+            />
+          </ListItem>
+          {userBookings.map((user) => {
+            const isAbsent = absentees?.includes(user.customer_id);
```

Comment:
This can be written directly into the component i.e.
```jsx
<UserAttendance 
  {...otherProps} 
  isAbsent={absentees?.includes(user.customer_id)} 
/>
```
And then you can return jsx diractly form the mapping function as `() => <UserAttendance />` without the unnecessary function block `{}`